src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -481,12 +483,24 @@ export const isDisabilityPtsd = disability => {
     return false;
   }
 
-  const loweredDisability = disability.toLowerCase();
-  return PTSD_MATCHES.some(
-    ptsdString =>
-      ptsdString.includes(loweredDisability) ||
-      loweredDisability.includes(ptsdString),
-  );
+  const strippedDisability = disability.toLowerCase().replace(/[^a-zA-Z]/g, '');
+
+  return PTSD_MATCHES.some(ptsdString => {
+    const strippedString = ptsdString.replace(/[^a-zA-Z]/g, '');
+    if (strippedString === strippedDisability) {
+      return true;
+    }
+
+    // does the veteran's input contain a string from our match list?
+    if (strippedDisability.includes(strippedString)) {
+      return true;
+    }
+
+    return (
+      fastLevenshtein.get(ptsdString, strippedDisability) <
```

Comment:
I think the one that's failing unexpectedly is because we're comparing the non-stripped `ptsdString` to the `strippedDisability`.
```js
fastLevenshtein.get('post traumatic stress disorder', 'plstrraumaticetressdisordre');
```
:point_up: has a distance of 8, but `Math.ceil('plst rraumatic etress disordre'.length * 0.25) === 7`

Reply:
doh!